proxy/src/test/java/com/wavefront/agent/listeners/tracing/JaegerPortUnificationHandlerTest.java

```diff
@@ -0,0 +1,195 @@
+package com.wavefront.agent.listeners.tracing;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.wavefront.agent.channel.NoopHealthCheckManager;
+import com.wavefront.agent.handlers.MockReportableEntityHandlerFactory;
+import com.wavefront.agent.handlers.ReportableEntityHandler;
+import com.wavefront.sdk.entities.tracing.sampling.RateSampler;
+import io.jaegertracing.thriftjava.Batch;
+import io.jaegertracing.thriftjava.Log;
+import io.jaegertracing.thriftjava.Process;
+import io.jaegertracing.thriftjava.Tag;
+import io.jaegertracing.thriftjava.TagType;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.http.*;
+import org.apache.thrift.TSerializer;
+import org.easymock.EasyMock;
+import org.junit.Test;
+import wavefront.report.Annotation;
+import wavefront.report.Span;
+import wavefront.report.SpanLog;
+import wavefront.report.SpanLogs;
+
+import static org.easymock.EasyMock.createNiceMock;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.expectLastCall;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.reset;
+import static org.easymock.EasyMock.verify;
+
+/**
```

Comment:
add a brief description of what these tests are doing?

Reply:
added a generic description